company_today/readme/DESCRIPTION.rst

```diff
@@ -0,0 +1,24 @@
+Store today's date on the company model. A cronjob makes sure this field stays
+up-to-date.
+
+The use-case for this module is as follows. Imagine you have two regular fields
+and a computed field::
+
+    amount = fields.Monetary()
+    payment_date = fields.Date()
+    cost_per_day = fields.Monetary(compute="_compute_cost_per_day")
```

Comment:
A question I had here was: in your real-world examples where you use this module, are these non-stored fields like in this example, or stored fields? Because I can imagine the use case for stored fields, but for non-stored the value is generally recomputed on the fly so it would hold to just use `fields.Date.today()`.

Reply:
Ah, yes, this concerns stored fields. I believe the use-case is to be able to sort on those computed-and-stored fields.